<!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML//EN">
<HTML>
<head><title>AFGC Website</title></head>        
<BODY  TOPMARGIN="10
"LEFTMARGIN="0" MARGINHEIGHT="0" MARGINWIDTH="0" BGCOLOR="#ffffff" >
<table width="560" border="0" cellspacing="1" cellpadding ="20">

<tr valign="top">
<td colspan="2" align="center">
<table width="460" border="0" cellspacing="1" cellpadding ="0"><td>
<a href="site1L.htm">
<IMG SRC="../images/FirstTabsAfgc.gif"WIDTH="115" HEIGHT="31" BORDER="0"></a>
</td>
<td>
<a href="site2L.htm">
<img  src="../images/FirstTabsAltcopyMicro.gif" WIDTH="115" HEIGHT="31" BORDER="0"></a>
</td>
<td>
<a href="site3L.htm">    
<img  src="../images/FirstTabsKnockouts.gif" WIDTH="115" HEIGHT="31" BORDER="0"></a>
</td>
<td>
<a href="site4L.htm">    
<img  src="../images/FirstTabsResources.gif" WIDTH="115" HEIGHT="31" BORDER="0"></a>
</td>
</tr>
<tr><td colspan=4 valign="top"bgcolor="FFFFCC" align="center"><B><br> <a href="site2Cycle1L.htm">ACCEPTED PROPOSALS</a>&nbsp;&nbsp;&nbsp;&nbsp;
<a href="site2RnaL.htm">PROTOCOLS</a>&nbsp;&nbsp;&nbsp;&nbsp;<a href="site2StatL.htm">STATISTICS</a></B></td></tr></table>
</td>


  <td rowspan="3" valign="top"><a href="../index.html"><center><img src="../images/arab4"><br>AFGC</center></a></td>
</TD>

</tr>

<tr><td  width="450"><h2><br>SERVICE FACILITY & ARRAY
INFORMATION</h2><br><h3><font color=red>
Updated August 1, 2002. The services have been terminated as of November 30, 2001.</font></h3>
<li><a href ="#Organization">Organization and Equipment</a>
<li><a href ="#Origin">Origin of DNAs</a>
<li><a href ="#DNAs">Number of DNAs on AFGC Microarrays</a>
<li><a href ="#Replicates">Replicates</a>

<p align="justify" colspan=4><basefont size=3>Please see <a
href="site4L.htm">Resources Link</a> for suggested reading and general
information about gene expression profiling by DNA microarray
analysis.</p>
<a name="Organization">
<br><h3>Organization and Equipment</h3><p align="justify">We operate as a single facility with two sites: MSU and Carnegie. Dr. Ellen Wisman at MSU will coordinate experiments to be done at the two sites with Dr. Shauna Somerville, who leads the AFGC Microarray team at the Carnegie. Both sites have identical equipment. The gridding robots were purchased from
<a href="http://www.genemachines.com">Gene Machines</a> and are similar to that
developed in the <!-- [[ a href ]] -->Brown lab. <a
href="http://www.arrayit.com/">Telechem</a> tips are used by the robots to
spot the DNA. Hybridization signals are detected with a <a
href="http://lifesciences.perkinelmer.com/areas/microarray/scanex.asp">Scan
Array 3000</a> or 4000 or an <a href="http://www.axon.com/GN_Genomics.html">Axon</a> 4000B Scanner.
</p><br></td></tr>

<tr><td><a name="Origin"><h3>Origin of DNAs</h3><p align="justify"><basefont size=3>
Most DNAs on AFGC arrays were prepared by PCR amplification of cDNA
  inserts from a nonredundant set of MSU EST clones. Please note that the
  DNA preparations used as PCR templates were from the same tubes used for
  the original EST sequencing performed by Tom Newman [Newman et al., 1994,
  Genes Galore: A summary of methods for accessing results from partial
  sequencing of anonymous Arabidopsis cDNA clones. Plant Physiol.,
  106:1241-1255]. This unique resource, kindly made available to the AFGC
  by Tom Newman, has allowed us to avoid handling errors often associated
  with large gene sets. A random selection of approximately 1,100 of the
  EST clones used for arrays were recently sequenced from AFGC stocks and
  99% matched their original designation (see User's Guide: <a
  href="SequenceVerL.htm">Sequence Verification</a>. Please note that the
  AFGC microarray DNAs are not derived from the Arabidopsis Biological
  Resource Center (ABRC) stocks. However, all clones can be requested from
  <a href="http://aims.cps.msu.edu/aims/">ABRC</a>. Users are encouraged
  to confirm the identity of requested clones upon receipt. A small
  fraction of DNA clones amplified for AFGC arrays were clones supplied by
  users. The Y2001 microarray arrays also contain 3,000 amplicons
  prepared by PCR amplification from genomic DNA with gene-specific primers.
</p><br></td></tr>

<tr><td><a name="DNAs"><h3>Number of DNAs on AFGC Arrays</h3>
<p align="justify"><basefont size=3> The Year 2000 array produced by the
AFGC contains DNAs representing about 11,000  ESTs from the
MSU collection representing approximately 7,500 genes. The Year 2001 array
consists of about 14,000 unique elements.</p><br></td></tr>

<tr><td colspan="2"align="justify"><a name="Replicates"><h3>Replicates</h3><p align="justify"><basefont size=3>
In the past, the AFGC has offered two microarrays for each experiment to
be used as biological replicates and/or as technical replicates of the
same two-sample comparison. In cycles 4 & 5, investigators had the option of requesting up to four microarray slides for each experiment (Note: 2 RNA samples are required for each microarray).
<p align="justify">

Both biological variation in the plant materials and technical variation arising from the handling steps (printing variation, labeling variation, hybridization variation) will contribute noise to the intensity measurements associated with each spot or gene. AFGC has observed that using technical replicates with a "reverse labeling" strategy can substantially reduce false positive results. Biological replicates are also very important to define genes that vary due to biological variation.  
<p align="justify">

The number of replications will depend heavily on the type of experiment and the goals of the experimenter. For example, our experience has shown that up to 25% of the genes in the array differ in expression by more than 2-fold in comparisons of different tissues.   In such comparisons, false positives represent a very minor proportion of the differentially expressed genes.  However, in some comparisons of mutant to wild-type, less than one percent of the genes on the array show a differential expression of 2-fold.  Although in both experiments, the number of false positives may be the same, the conclusions of the mutant/wild-type comparison may be much more severely impacted.
<p align="justify">

An additional analysis of the impact of replications on microarray data is available in the paper:
Lee ML, Kuo FC, Whitmore GA, Sklar J. (2000)   Importance of replication in microarray gene expression studies: statistical methods and evidence from repetitive cDNA hybridizations.  Proc Natl Acad Sci U S A. 97:9834-9.


</td></tr>
</table>
</table></td></tr></table></td>
</body></html>
