<!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML//EN">
<HTML>
<head><title>AFGC Website</title></head>        
<BODY  TOPMARGIN="10
"LEFTMARGIN="0" MARGINHEIGHT="0" MARGINWIDTH="0" BGCOLOR="#ffffff" >
<table width="560" border="0" cellspacing="1" cellpadding ="0">
<tr>
  <td>
<table width="460" border="0" cellspacing="1" cellpadding ="0" valign="top"><td>
<a href="site1L.htm">
<IMG SRC="../images/FirstTabsAltcopyAfgc.gif"WIDTH="115" HEIGHT="31" BORDER="0"></a>
</td>
<td>
<a href="site2L.htm">
<img  src="../images/FirstTabsMicroarray.gif" WIDTH="115" HEIGHT="31" BORDER="0"></a>
</td>
<td>
<a href="site3L.htm">    
<img  src="../images/FirstTabsKnockouts.gif" WIDTH="115" HEIGHT="31" BORDER="0"></a>
</td>
<td>
<a href="site4L.htm">    
<img  src="../images/FirstTabsResources.gif" WIDTH="115" HEIGHT="31" BORDER="0"></a>
</td>
</tr>
<tr><td colspan=4 valign="top"bgcolor="FFFFCC" align="center"><B><a href="site1PlantL.htm">PLANT SPECIFIC GENES</a>&nbsp;&nbsp; AND &nbsp;&nbsp;
  <a href="site1StaffL.htm">STAFF</a></B>
  </td></tr></table>


 <td rowspan="4" valign="top"><center><a href="../index.html"><img src="../images/arab4"><br>AFGC</a></center></td>
</TD></tr>

  <tr><td align="center"><font size="4"><b><br>OVERVIEW</b></font>
    </td></tr>

<tr><td align="center"><b><font color =red><H2>Updated August 1, 2002</h2></b></font></td></tr>
<tr><td width="450">
<h3 align="center"> A Functional Analysis of the Arabidopsis Genome via
Global Gene Expression Analysis and Gene Disruption  <P>(January 1, 1999 toDecember 31, 2002)</h3></td></tr></table>
<table width="560" cellpadding ="20">
<tr><td><p align="justify">
<basefont size=3>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; 
In the near future, the complete sequence of the Arabidopsis genome will
be known and this together with cDNA sequencing will lead to the
identification of nearly all Arabidopsis genes. Most of these genes will
have homologs in other plants and many will have unknown function. The
overall goal of this project is to provide plant biologists with a set of
powerful tools that can be used in their efforts to understand the
function and interrelationships of the 20,000 or more genes in this and
other plants.
</td>
</tr>
<tr>
<td>
  <p align="justify">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Our project has three components: technology development,
service to the community, and biological application. It is being executed
by the Arabidopsis Functional Genomics Consortium (AFGC) which is comprised of a <a href="site1StaffL.htm">team</a> of investigators from
four institutions: Michigan State University (Pam Green, PI, and Ken
Keegstra and John Ohlrogge, co-PIs), University of Wisconsin, Madison
(Mike Sussman and Rick Amasino, co-PIs), Yale University (Steve
Dellaporta, co-PI), Carnegie Institute of Washington at Stanford University (Shauna Somerville and Mike Cherry, co-PIs).</p></td></tr>
<tr><td>
<p align="justify"><basefont size=3>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
Two synergistic technologies to investigate gene function are being
developed and provided at service facilities. The
<a href="site2L.htm">microarray facility</a>
 will assemble DNA microarrays of many of the genes of Arabidopsis. By
hybridizing these arrays with probes corresponding to mRNA from different
tissues and organs, or from plants subjected to different stimuli, global
gene expression patterns can be investigated. We are also setting up
database and software support to facilitate sophisticated searching and
comparisons among these global expression patterns. The community will be
provided access through the WWW to AFGC microarray data within the
Stanford Genetics Microarray database. In parallel with our microarray
experiments, our second technology development effort has been to optimize
high throughput screening of Arabidopsis plants for T-DNA insertion
mutations ("gene knockouts"). This has enabled the establishment of a <a
href="site3L.htm">gene knockout facility</a>. We are also testing
new ways to further maximize the efficiency of identifying gene knockouts
that will rely on the use of junction fragments. Both the gene knockout
facility and the microarray facility comprise the service component of our
project. These technologies will be made available to the academic plant
biology community beginning in year two of the project (during the year
2000) and with "no strings attached" with respect to intellectual
property. The coordinator of the DNA Microarray facility is
Ellen Wisman and the coordinator of
the Knockout Facility is Sandra
Austin-Phillips.</p></td></tr>
<tr><td>
<p align="justify"> <basefont size=3>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The biological
application component of this proposal will focus on enhancing our
understanding of<a href="site1PlantL.htm"> plant-specific genes</a> of
unknown function, but it will generate a large amount of data on the other
Arabidopsis genes as well. Plant-specific genes, i.e. genes found in
Arabidopsis and other plants but not in yeast, bacterial, or animal
systems, are of great interest because they may be involved in
plant-specific processes such as plant-pathogen interactions, cell-wall
biosynthesis, and others. Using our bioinformatics expertise, we will identify and incorporate onto the AFGC microarrays, many of the
plant-specific genes of Arabidopsis. We will then examine their expression
in the major organs of Arabidopsis and under at least 20 different
conditions. Knockout mutations will be identified in a number of these
genes and the mutant plants will be examined for altered phenotypes. The
microarray data for all Arabidopsis genes studied in this project will be
available through the WWW from a database specifically designed for that
purpose. Summaries of the plant-specific gene data will also be provided
which will be hyperlinked, via the WWW, back to data within the microarray
database. Seed from the mutants will be distributed through ABRC. This
will allow members of the community to formulate and test hypotheses about
the function of these plant-specific genes. </p></td></tr>
<tr><td>
<p align="justify"><basefont size=3>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; Taken together, the information and materials generated from this project should help advance plant genome research to the next level. Just
as the Arabidopsis EST project benefitted ongoing research in a variety of
plant systems, the impact of this project is expected to extend far beyond Arabidopsis. This is particularly true for the information obtained about
plant-specific genes, many of which will be of general significance to
crop plants and other plants of commercial value.
</p></td></tr>
</table>
</body> </html>












