<!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML//EN">
<HTML>
<head><title>AFGC Website</title></head>        
<BODY  TOPMARGIN="10
"LEFTMARGIN="0" MARGINHEIGHT="0" MARGINWIDTH="0" BGCOLOR="#ffffff" >

<table width="560" border="0" cellspacing="1" cellpadding ="5">
<tr>
 <td>
     <table width="460" border="0" cellspacing="1" cellpadding ="0">
       <td>
 <a href="site1L.htm">
 <IMG SRC="../images/FirstTabsAltcopyAfgc.gif"WIDTH="115" HEIGHT="31" BORDER="0"></a>
</td>
<td>
<a href="site2L.htm">
<img  src="../images/FirstTabsMicroarray.gif" WIDTH="115" HEIGHT="31" BORDER="0"></a>
</td>
<td>
<a href="site3L.htm">    
<img  src="../images/FirstTabsKnockouts.gif" WIDTH="115" HEIGHT="31" BORDER="0"></a>
</td>
<td>
<a href="site4L.htm">    
<img  src="../images/FirstTabsResources.gif" WIDTH="115" HEIGHT="31" BORDER="0"></a>
</td>
</tr>
</td>
<td colspan=4 valign="top"bgcolor="FFFFCC" align="center"><B><a href="site1PlantL.htm">PLANT SPECIFIC GENES</a>&nbsp;&nbsp; AND &nbsp;&nbsp;<a href="site1StaffL.htm">STAFF</a></B></td></tr></table>
<td rowspan="3" valign="top"><a href="../index.html"><center><img src="../images/arab4"><br>AFGC</center></a></td>
</tr>

<tr> 
<td  valign="top" ><font size="4">
<center><br> <b>AFGC PLANT-SPECIFIC GENES</b></center></font>
</TD>
</tr>
<tr><td align =center><b><h2><font color=red>Updated August 1,
2002</h2><br><h4><font color=red>Note 1:  A <a href="http://www.bch.msu.edu/pamgreen/PS_project/front.html">Plant-Specific Genes website</a> has been developed
by the Michigan State University group (Pam Green, Ken Keegstra, John Ohlrogge)
and will be maintained beyond the termination of the AFGC project.</h4></font></td></tr></table>
<table width="560" border="0" cellspacing="1" cellpadding ="20">
<tr><td>  
<p align="justify"><basefont size=3>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
One objective of our project is to further our understanding of plant-specific genes, that is genes found in multiple plants but not other organisms. Our rationale for this emphasis is threefold. First, plant-specific genes have a high probability of functioning in plant-specific processes. Second, information about plant-specific genes should contribute to diverse areas of plant biology. Third, because plant-specific genes by definition exist in multiple plants, examining such genes in more detail will provide information and materials that should be useful for researchers working with many different plant species.</p></td></tr>


<tr><td>
<p align="justify"><basefont size=3>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
This aspect of our project will focus on about 50 plant-specific genes chosen as described below. The expression of these genes will be examined under a variety of conditions using DNA microarrays. We will also screen for knockout mutations in these genes. Summaries of expression information and any visible phenotypes associated with the Knockout (KO) mutations for each plant-specific gene will be published on TAIR and/or this website for community access. Links to other databases that describe homologs of these genes in other plant species will be included.<br></p></td></tr>
<tr><td>
<p align="justify"><basefont size=3>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;    
To identify plant-specific genes, the genes of Arabidopsis will be compared to genes of other plants and those of other organisms. A list of plant-specific genes, found in multiple plants but not other organisms, will be identified. Genes of known function will be removed from the list and the remaining genes of unknown function will be classified into the three categories below. The 50 plant-specific genes that will be the focus of AFGC studies will be selected from among the genes in these categories.</p></td></tr>
<tr><td>
<p align="justify"><br><h3><b>Categories of Plant-Specific Genes:</h3></b><p>
I. Plant-specific genes with no homology that suggests function.<br><br>
II. Genes with suspected enzymatic, structural, or process
association based on domain homologies or known location of gene product,
but whose biological role is unknown.<br><br>
III. Genes with homology to genes of known or suspected function that have
in addition a plant-specific domain (i.e., a domain found in multiple
plants, but not other organisms), of unknown function. Some genes may
fulfill criteria of both categories I and II.<br><br>
<p>The <a href="http://www.bch.msu.edu/pamgreen/PS_project/front.html">plant-specific genes website</a> will be maintained by the group at
Michigan State University (John Ohlrogge, Ken Keegstra).<BR>

</p></td></tr>

<tr><td><br>
</td></tr>

<th><h2><align="center"> AFGC Experiments</h2></th>
<tr><td>
<p align="justify"><basefont size=3>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
AFGC Personnel at Michigan State University and Carnegie Institution of
Washington (CIW) will carry out several DNA microarray experiments to
provide general information about plant-specific genes and other genes on
AFGC arrays. These experiments will include expression studies of the
major organs of Arabidopsis such as leaf, stem, root and flowers. In
addition, at least twenty other array experiments will be performed to
examine certain aspects of development (e.g., some stages of leaf, stem
and flower development) and the response of Arabidopsis to selected
stimuli (e.g., light, carbon dioxide, wounding, methyl jasmonate and
circadian rhythms). All resulting data will be made publicly available
through the  <A
HREF="http://genome-www4.Stanford.EDU/MicroArray/SMD/">Stanford Microarray
Database.</a></p><br> <br>
<p><b>Microarray-Related Papers by AFGC Personnel</B>

<ul>Note: The Experiment ID for data sets in the Stanford Microarray Database are given following the related publication. 
<p><li>Ewing, R.M. and J.M. Cherry. 2001. Visualization of expression clusters using Sammon's non-linear mapping. Bioinformatics 17: 658-659.
<li>Finkelstein, D., R. Ewing, J. Gollub, F. Sterky, et al. 2000. Iterative linear regression by sector. Presentation at CAMDA '00 Conference on the Critical Assessment of Techniques for Microarray Data Analysis, December 18-19, 2000, Duke, N.C.
<li>Finkelstein, D., R. Ewing, J. Gollub, F. Sterky, et al. 2002. Microarray data quality analysis: lessons from the AFGC project. Plant Molecular Biology 48: 119-131.
<li>Girke, T., J. Todd, S. Ruuska, J. White, et al. 2000. Microarray analysis of developing Arabidopsis seeds. Plant Physiology 124: 1570-1581. 
<li>Jackson, R.B., C.R. Linder, M. Lynch, M. Purugganan, et al. 2002. Linking molecular insight and ecological research. Trends in Ecology and Evolution 17: 409-414.
<li>Monte, D. and S. Somerville. 2002. Protocol 3: Isolation of total RNA from plant tissue using TRIzol. In: DNA Microarrays: A Molecular Cloning Manual (D. Botwell and J. Sambrook, eds.). Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY (in press)
<li>Perez-Amador, M.A., P. Lidder, M.A. Johnson, J. Landgraf et al. 2001. New molecular phenotypes in the <i>dst</i> mutants of Arabidopsis revealed by DNA microarray analysis. Plant Cell 13: 2703-2717.
<li>Ramonell, K.M. and S. Somerville. 2002. The genomics parade of defense responses: To infinity and beyond. Current Opinion in Plant Biology 5: 291-294. 
<li>Ramonell, K.M., B. Zhang, R.M. Ewing, Y. Chen, et al. 2002. Microarray analysis of chitin elicitation in <i>Arabidopsis thaliana</i>. Molecular Plant Pathology (in press). SMD Expt ID: 10615, 10616, 10618, 10619, 10644, 10645, 10648, 10649, 10650, 10961, 10962, 10963.
<li>Ruuska, S.A. and J.B. Ohlrogge. 2001. Protocol for small-scale RNA isolation and transcriptional profiling of developing Arabidopsis seeds. Biotechniques 31: 752.
<li>Ruuska, S.A., T. Girke, C. Benning and J.B. Ohlrogge. 2002. Contrapuntal networks of gene expression during Arabidopsis seed filling. Plant Cell 14: 1191-1206. 
<li>Schaffer, R., J. Landgraf, M. PerezAmador and E. Wisman. 2000. Monitoring genome-wide expression in plants. Current Opinion in Biotechnology 11: 162-167. 
<li>Schaffer, R., J. Landgraf, M. Accerbi, V. Simon et al. 2001. Microarray analysis of diurnal and circadian-regulated genes in Arabidopsis. Plant Cell 13: 113-123. SMD Expt ID: 2344, 2359, 2361-2368, 3242, 10177-10180. 
<li>Wisman, E. and J. Ohlrogge. 2000. Arabidopsis microarray service facilities. Plant Physiology 124: 1468-1471.
<li>Wu, S.-H., K. Ramonell, J. Gollub and S. Somerville. 2001. Plant gene expression profiling with DNA microarrays. Plant Physiology Biochemistry 39: 917-926.
 

</td></tr>
</table></td></td></tr>
</table>
</body> </html>





